platforms/anchor-platform/admin-guide/sep6/configuration.mdx

```diff
@@ -197,6 +209,8 @@ SEP1_TOML_VALUE=/home/dev.stellar.toml
 
 SEP6_ENABLED=true
 SEP6_DEPOSIT_INFO_GENERATOR_TYPE=none
+SEP6_MORE_INFO_URL_BASE_URL=http://example.com
```

Comment:
These are optional, right? Should we mark them as such?

Reply:
Yes, I believe I mentioned at above that they are optional url.